apps/api/src/billing/routes/sign-and-broadcast-tx/sign-and-broadcast-tx.router.ts

```diff
@@ -11,7 +11,13 @@ export const SignTxRequestInputSchema = z.object({
     messages: z
       .array(
         z.object({
-          typeUrl: z.string(),
+          typeUrl: z.enum([
+            "/akash.deployment.v1beta3.MsgCreateDeployment",
```

Comment:
Not an issue, but we'll need to be aware when there's new types or network upgrades. I think zod throw an error if we send a value outside of the enum.

Reply:
Exactly, I think we discussed this. It's unwanted to misuse an api. So I'd prefer any other values are rejected. If there's an upgrade we'll do it as well